src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -484,7 +484,7 @@ class TaskStatusBarItem extends Themable implements IStatusbarItem {
 		element.appendChild(labelElement);
 
 		let label = new OcticonLabel(labelElement);
-		label.title = nls.localize('runningTasks', "Show Running Tasks");
+		label.title = nls.localize('runningTasks', "Show Running Task Outputs");
```

Comment:
But the action itself only show the output directly if there is only one task running. Otherwise it will show a list of tasks to select from. To bring this more in line with the rest of our commands we might want to name it Go to running task... We don't use Show somewhere else and then it is more clear that users select a task and will be guide to their output or terminal.